test/test_torch.py

```diff
@@ -3433,15 +3433,15 @@ def check_order(a, b):
             self.assertTrue(check_order(mxx[j][k - 1], mxx[j][k]),
                             'torch.sort ({}) values unordered for {}'.format(order, task))
 
-        seen = set()
+        seen = []
         indicesCorrect = True
         size = x.size(x.dim() - 1)
         for k in range(size):
-            seen.clear()
+            seen = []
             for j in range(size):
                 self.assertEqual(x[k][ixx[k][j]], mxx[k][j],
                                  'torch.sort ({}) indices wrong for {}'.format(order, task))
-                seen.add(ixx[k][j])
+                seen.append(ixx[k][j])
             self.assertEqual(len(seen), size)
```

Comment:
Well this is always trivially true, because that's how many `appends` you're doing. You probably want to have a `set()` of `.item()`s.